src/marine/godae_profile2ioda.py

```diff
@@ -169,25 +169,29 @@ def __init__(self, filename, date, varDict, obsList):
             ("datetime", "string")
         ]
 
-        self.AttrData = {
+        self.GlobalAttrs = {
             'odb_version': 1,
             'date_time_string': self.date.strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
The PR looks good to me. On minor thing: I think it was decided to remove the date_time _string here, not sure if it matters right now or not. I forgot what the reason was but @rhoneyager or @CoryMartin-NOAA remember may be.

Reply:
@kbhargava thanks for the review! I'll remove the date_time_string attribute.